src/openforms/formio/rendering/nodes.py

```diff
@@ -230,7 +230,10 @@ def __iter__(self) -> Iterator["ComponentNode"]:
             return
 
         # in export mode, only emit if the component is not a layout component
-        if self.mode != RenderModes.export or not is_layout_component(self.component):
+        if self.mode != RenderModes.export or (
+            not is_layout_component(self.component)
+            and self.component["type"] != "editgrid"
```

Comment:
@sergei-maertens I do have to add this check here explicitly after I added the `editgrid` check to `is_layout_component`

Reply:
yeah we shoved editgrid in there in a weird way, I'm glad it's not worse than this